exampleSite/content/about/footer.md

```diff
@@ -5,10 +5,13 @@ date = "2016-09-07"
 lastmod = "2017-09-07"
 #background = ""
 
-link_title = "Link Title"
-logo_title = "Logo Title"
-logo = "logo.svg"
-logo_text = "Logo Subtext"
+[[link]]
+    title = "Link Title"
```

Comment:
Not sure about that part. It feels too much nesting without the actual need. Do we expect "link" to have additional sub items in the future? Might link_title just have been a bad key name?
What is link_title actually standing for?

Reply:
Not really "link" has just one sub item there.  I think we can modify the key name.
"link-title" there stands for the title for the links in the footer. The other links are generated from the config.toml file.

Reply:
"menu_title" or something?

Reply:
Yeah without the nesting.